hack/local-up-cluster.sh

```diff
@@ -150,6 +151,11 @@ if [ "$(id -u)" != "0" ]; then
     echo "WARNING : This script MAY be run as root for docker socket / iptables functionality; if failures occur, retry as root." 2>&1
 fi
 
+if [ -z "$CLOUD_PROVIDER" ]; then
+    ENABLE_HOSTPATH_PROVISIONER=true
+    echo "WARNING : The hostpath provisioner is used only for development and testing only and WILL NOT WORK in a multi-node cluster."
```

Comment:
This indicates that the hostpath provision is used for development, but doesn't actually say that the script is taking the action of enabling it.